dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/LevelOrderQueueItem.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dependencies;
+
+import java.util.Objects;
+
+/**
+ * The item in the queue for the level order graph traversal when constructing {@link
```

Comment:
Does anything in this class imply or require that it's level-order? Or is it just a queue item?

Reply:
No, it does not need to be level-order. Removed the word "level order".